flake8_dcos_lint/check_rules.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+from collections import namedtuple
+
+CheckRegex = namedtuple("CheckRegex", "regex code reason")
+
+
+regex_rules = [
+    CheckRegex(re.compile(r'assert\s(\w+\.ok)$'), "D001", "Invalid assertion of type assert <symbol>.ok found.")
```

Comment:
Can we add a comment here with the rationale for this rule? It'll at least be something to point to when someone tries to add an `ok` attribute to their class and runs into this.